base/sparse/sparsevector.jl

```diff
@@ -122,7 +122,8 @@ end
 
 Create a sparse vector `S` of length `m` such that `S[I[k]] = V[k]`.
 Duplicates are combined using the `combine` function, which defaults to
-`+` if it is not provided.
+`+` if no combine argument is provided except for Boolean elements
```

Comment:
"if no `combine` argument is provided, unless the elements of `V` are Boolean in which case combine defaults to `|`" ?